### PR TITLE
fix: TR-3035. Mirror "Skip" button

### PR DESCRIPTION
**Pull request by svoronuk**
Related to: https://oat-sa.atlassian.net/browse/TR-3035

How to test: better test it in terms of https://github.com/oat-sa/extension-tao-testqti/pull/2136

### CODE REVIEW

File: scss/inc/rtl/_action-bar.scss [hunk 1]
```diff
@@ -76,5 +76,8 @@
 
       }
     }
+    .icon-external{
+      transform: scale(-1, 1);
```
Comment (oatymart) [inline scss/inc/rtl/_action-bar.scss:80]: It is flipped, but does it need padding change as well?
Screenshot 2021-11-30 at 15 24 03
Screenshot 2021-11-30 at 15 24 13

Look at the SCSS above, all the other icons were done in a block with padding updated.

Comment (svoronuk) [reply scss/inc/rtl/_action-bar.scss:80]: Good catch

**Review by oatymart — APPROVED** [review]
Code looks better now!

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [ ] Documentation is updated according to changes (if applicable)

**Review by olga-kulish — APPROVED** [review]
Works well

**Pull request merged**